src/main/java/io/cdap/plugin/http/common/http/MessageFormatType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.http.common.http;
+
+import io.cdap.plugin.http.common.EnumWithValue;
+
+/**
+ * An enum which represent a type of message format of an HTTP request body.
+ */
+public enum MessageFormatType implements EnumWithValue {
+
+  JSON("JSON"),
+  FORM("FORM"),
+  CUSTOM("CUSTOM");
```

Comment:
there isn't a need for these values, can just use a plain enum. `Enum.name()` is the same thing as these values.

Reply:
```java
public enum MessageFormatType implements EnumWithValue {

  JSON,
  FORM,
  CUSTOM;

  @Override
  public String getValue() {
    return name();
  }

  @Override
  public String toString() {
    return this.getValue();
  }
}
```